syndeo-template/src/test/java/com/google/cloud/syndeo/common/SyndeoTemplateLaunchTest.java

```diff
@@ -37,11 +37,31 @@ public class SyndeoTemplateLaunchTest {
           + "  }\n"
           + "}";
 
+  private static final String BQ_TO_BQ_CONFIG =
+      "{ \"source\": "
+          + "{ \"urn\": \"bigquery:read\", "
+          + "\"configurationParameters\": { \"tableSpec\": \"dataflow-syndeo:test_dataset.kafka_table\" } }, "
+          + "\"sink\": { \"urn\": \"bigquery:write\", "
+          + "\"configurationParameters\": { \"tableSpec\": \"dataflow-syndeo:test_dataset.source_table\" } } }";
+
   @Test
   public void testBuildWithListOfValuesBigTable() throws JsonProcessingException {
     ObjectMapper om = new ObjectMapper();
     JsonNode btconfig = om.readTree(BIGTABLE_SINK_CONFIG);
     SyndeoTemplate.buildFromJsonConfig(btconfig);
     // TODO(pabloem): Add more checks
   }
+
+  @Test(expected = IllegalArgumentException.class)
+  public void testBuildWithBQToBQConfigBadSink() throws JsonProcessingException {
+    SyndeoTemplate.buildFromJsonPayload(BQ_TO_BQ_CONFIG);
+  }
+
+  @Test
+  public void testBuildWithBQToBQConfig() throws JsonProcessingException {
+    // TODO(pabloem): Replace this with
+    // "beam:schematransform:org.apache.beam:bigquery_storage_write:v1" for Beam  2.45.0.
+    SyndeoTemplate.buildFromJsonPayload(
+        BQ_TO_BQ_CONFIG.replace("bigquery:write", "schemaIO:bigquery:write"));
```

Comment:
Just to understand, it would fail without the replacement string because this transform is in beam and not in templates (unlike the read schema transform)?

Reply:
the `urn` is set by the transform itself. We just don't happen to have an implementation that returns `bigquery:write` as its URN. But yes, you're correct otherwise.